platform-emf/src/main/java/com/softicar/platform/emf/form/scope/EmfFormViewScopeAction.java

```diff
@@ -0,0 +1,90 @@
+package com.softicar.platform.emf.form.scope;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.core.utils.CastUtils;
+import com.softicar.platform.common.io.resource.IResource;
+import com.softicar.platform.db.sql.field.ISqlForeignRowField;
+import com.softicar.platform.dom.elements.popup.manager.DomPopupManager;
+import com.softicar.platform.emf.EmfI18n;
+import com.softicar.platform.emf.EmfImages;
+import com.softicar.platform.emf.action.IEmfCommonAction;
+import com.softicar.platform.emf.attribute.field.foreign.row.IEmfForeignRowAttribute;
+import com.softicar.platform.emf.form.IEmfFormBody;
+import com.softicar.platform.emf.form.popup.EmfFormPopup;
+import com.softicar.platform.emf.mapper.IEmfTableRowMapper;
+import com.softicar.platform.emf.permission.EmfMappedPermission;
+import com.softicar.platform.emf.permission.EmfPermissions;
+import com.softicar.platform.emf.permission.IEmfPermission;
+import com.softicar.platform.emf.predicate.EmfPredicate;
+import com.softicar.platform.emf.predicate.IEmfPredicate;
+import com.softicar.platform.emf.table.IEmfTable;
+import com.softicar.platform.emf.table.row.IEmfTableRow;
+
+public class EmfFormViewScopeAction<R extends IEmfTableRow<R, ?>, F extends IEmfTableRow<F, ?>> implements IEmfCommonAction<R> {
```

Comment:
I think this should be renamed to `EmfViewScopeAction`, resembling `EmfEditAction`.

Reply:
![image](https://user-images.githubusercontent.com/90852097/179186240-df517a91-630e-43cd-9b44-1e012788cf45.png)
Rationale:
  - `EmfForm`-Actions are common action, only accessible in the `EmfForm`
  - The `Edit` and `Log` actions also are `ManagementActions`
Therefore, the current name is the correct one

Reply:
I see